content/js3/blocks/using-fetch/index.md

```diff
@@ -0,0 +1,53 @@
++++
+title = '🌐 Requesting from a server side API'
+headless = true
+time = 20
+facilitation = false
+emoji= '🧩'
+[objectives]
+1="List 4 preceding concepts of asynchronous programming in JavaScript"
+2="Identify unknown concepts still to be learned"
+3="Fetch data from a server side API"
++++
+
+So now we have these pieces of our giant concept map
+
+🗓️ we know that sending 📤 requests over a network takes time
+🧵 we know that we should not stop our program to wait for data
+📤 we know that we can send a request using `fetch()`
+🐕 we know that `fetch` is a 💻 client side 🧰 Web API
+🫱🏿‍🫲🏽 we know that `fetch` returns a `Promise`? (...this last one is pending)
```

Comment:
I suspect this is more confusing than it is funny, to our trainees? Maybe remove?

Reply:
the whole thing? or just the last line

It's good to recap what we know. I will remove jokes no jokes allowed I know

🧵 we know that we should not stop our program to wait for data
📤 we know that we can send a request using `fetch()`
🐕 we know that `fetch` is a 💻 client side 🧰 Web API
🫱🏿‍🫲🏽 we don't yet know that `fetch` returns a `Promise`, and we will investigate this next time